package.json

```diff
@@ -10,8 +10,8 @@
     "test": "IASQL_ENV=test jest -i",
     "coverage": "IASQL_ENV=test jest -i --coverage",
     "coverage:local": "IASQL_ENV=test export $(cat .env | xargs) && yarn coverage",
-    "start": "if [ \"$IASQL_ENV\" = \"\" ]; then IASQL_ENV=production yarn serve; elif [ \"$IASQL_ENV\" = \"local\" ]; then yarn start:dev; else yarn serve; fi",
```

Comment:
removing this breaks the `docker-compose:dev` script to start locally with hot reload. I'll see how can we could maintain it